terraform/gcp/modules/monitoring/prober/prober_alerts.tf

```diff
@@ -184,3 +184,37 @@ resource "google_monitoring_alert_policy" "prober_error_codes" {
   notification_channels = local.notification_channels
   project               = var.project_id
 }
+
+resource "google_monitoring_alert_policy" "prober_verification" {
+  alert_strategy {
+    auto_close = "604800s"
+  }
+
+  combiner = "OR"
+
+  conditions {
+    condition_threshold {
+      aggregations {
+        alignment_period   = "60s"
+        per_series_aligner = "ALIGN_SUM"
+      }
+
+      comparison      = "COMPARISON_GT"
+      duration        = "0s"
+      filter          = "resource.type = \"k8s_container\" AND metric.type = \"external.googleapis.com/prometheus/verification\" AND metric.labels.verified = \"false\""
+      threshold_value = "0"
+
+      trigger {
+        count   = "1"
+        percent = "0"
+      }
+    }
+
+    display_name = "Kubernetes Container - external/prometheus/verification"
+  }
+
+  display_name          = "API Prober: Verification failed at least once in the last 60s"
```

Comment:
Normally I would associate a prober failure to be relating to reliability rather than correctness. 

Could the display name be adjusted to something that is more obvious to oncall responders? I may not have the terminology right, but I was thinking something like:

`Rekor write correctness: verifier returned 'false' within the last 60s`

Reply:
Good point! I took your proposal and prefixed with `API Prober` (for consistency) and added a documentation section with a rewording.